pages/api/user/[userId].ts

```diff
@@ -0,0 +1,64 @@
+import { User } from '@prisma/client';
+import { NextApiRequest, NextApiResponse } from 'next';
+import { errorResponse, successResponse } from '../../../lib/http.response';
+import { prisma } from '../../../lib/prisma';
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (req.method === 'GET') {
```

Comment:
instead of if statements could you use switch statement?

Reply:
Ya I could and can I ask you Why?

Reply:
switch statement makes it cleaner imo. makes code less as compared to if statement.